src/main/java/frc/robot/SwerveModule.java

```diff
@@ -190,40 +187,40 @@ public double getRotPosition() {
         return getRotPositionRaw();
```

Comment:
A bug here? in many places we call getRotPosition() and expect radians... but looks like getRotPositionRaw() return degrees.

Reply:
Also dont create a separate function that has no logic except calling another function.

Reply:
I call this in a separate function in case we switch encoders and a new encoder ticks variable is required to multiply the rawRotPostion function by. Aadi explained the logic behind the getRotPosition() function in the previous comment